### DISCUSSION TITLE
Adopt SPEC 0?

### DISCUSSION THREAD

**Discussion by rcomer** [SciTools/iris#5697] [General]
There is now a note on the [NEP29 page](https://numpy.org/neps/nep-0029-deprecation_policy.html) stating that it has been superseded by [SPEC 0](https://scientific-python.org/specs/spec-0000/).

SPEC 0 has a shorter support window for python versions, so we could drop support for python3.9 already.  It also defines support windows for some of our other dependencies e.g. scipy and matplotlib.

**Comment by bjlittle** [comment]
I've already dropped support for py39 on `geovista` a while ago.

I felt that nep29 was a tad conservative.

But I guess you can only move as fast as your slowest dependency.
